model_compression_toolkit/exporter/model_wrapper/fw_agnostic/get_inferable_quantizers.py

```diff
@@ -32,18 +32,25 @@ def get_inferable_quantizers(node: BaseNode,
     Returns:
         weight_quantizers: A dictionary between a weight's name to its quantizer.
         activation_quantizers: A list of activations quantization, one for each layer output.
+        weight_values: A dictionary between a weight's name to its value. Relevant for positional weights only.
     """
 
     weight_quantizers = {}
     activation_quantizers = []
+    weight_values = None
```

Comment:
Can you init this with an empty dict and then you can skip the `if weight_values is None:` part later and just update the dictionary with a key-value? would it be a problem to return an empty dict instead of None?